zeek/ivre/passiverecon/ntlm.zeek

```diff
@@ -32,16 +32,22 @@ function _get_protocol_version(c: connection): string {
 }
 
 # Returns a string made from the list of protocols detected by Zeek
-function _get_source(c: connection): string {
+function _get_source(c: connection, src: port, proto: string &default=""): string {
     local protocols = vector();
     for (p in c$service) {
-        protocols += p;
+        if (p !in "GSSAPI, NTLM" && p != proto) {
```

Comment:
You are testing string "in", I'd rather have `p !in {"GSSAPI", "NTLM"}` (here, when p is `"G"`, the condition does not match and the protocol is not added).